### ISSUE TITLE
Deprecation warning from Shapely

### ISSUE THREAD

**Issue by jpswinski** [ICESat2-SlideRule/sliderule-python#95]
When running the region_of_interest.py utility, there are sometimes multiple deprecation warnings issued as below:

```
..../python3.8/site-packages/geopandas/array.py:258: ShapelyDeprecationWarning: The array interface is deprecated and will no longer work in Shapely 2.0. Convert the '.coords' to a numpy array instead.
  return GeometryArray(vectorized.points_from_xy(x, y, z), crs=crs)
```

**Comment by jpswinski** [comment]
Latest default installs of SlideRule no longer exhibit this warning as packages have been updated.
